homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js

```diff
@@ -0,0 +1,32 @@
+function UserInfo() {
```

Comment:
camelCase naming here please

Reply:
renamed, but eslint in vs code said me that require constructor names to begin with a capital letter (new-cap)